scripts/west_commands/runners/uf2.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2023 Peter Johanson <peter@peterjohanson.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+'''UF2 runner (flash only) for UF2 compatible bootloaders.'''
+
+from pathlib import Path
+from shutil import copy
+
+from runners.core import ZephyrBinaryRunner, RunnerCaps
+
+try:
+    import psutil  # pylint: disable=unused-import
+    MISSING_PSUTIL = False
+except ImportError:
+    # This can happen when building the documentation for the
+    # runners package if edtlib is not on sys.path. This is fine
+    # to ignore in that case.
+    MISSING_PSUTIL = True
+
+class UF2BinaryRunner(ZephyrBinaryRunner):
+    '''Runner front-end for bossac.'''
+
+    def __init__(self, cfg, board_id=None):
+        super().__init__(cfg)
+        self.board_id = board_id
+
+    @classmethod
+    def name(cls):
+        return 'uf2'
+
+    @classmethod
+    def capabilities(cls):
+        return RunnerCaps(commands={'flash'})
+
+    @classmethod
+    def do_add_parser(cls, parser):
+        parser.add_argument('--board-id', required=False, dest='board_id',
```

Comment:
`required=False` is redundant here; please remove

Reply:
Removed.